orbital-mechanics/TheRealLecture1/SpacecraftDynamicsWeek1r;$.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<p>Good, but minor formatting issues to bring it in line with other lectures</p><ul><li>Remove “Lecture 1” heading</li><li>All vectors are denoted by a bold face. This can be done using ‘\mathbf{}’. This is applied throughout the lecture. </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<pre class="ql-syntax" spellcheck="false"># Vectors (Brief Recap)
</pre><ul><li>Following with what Mitansh has mentioned, you can make this as the main title of the chapter. It is good as a first level heading.</li><li>This heading will also need to have its own markdown cell.</li><li>Lastly, you would need to add the list of authors below the title, for example:</li></ul>

<pre class="ql-syntax" spellcheck="false"># Vectors (Brief Recap)
Prepared by: [insert full name here](https://github.com/insert your username here) and [Angadh Nanjangud](https://www.angadhn.com/)
</pre>